### PR TITLE
CRE1-591 Adds grafana dashboards created with jsonnet

### PR DESCRIPTION
**Pull request by rodrmartinez**
- Adds all the [kubernetes-mixin](https://github.com/kubernetes-monitoring/kubernetes-mixin) Grafana Dashboards using jsonnet.
- Includes a `Makefile` and `Dockerfile` to recreate the dashboards when changed
- Uses `kustomization` to create all the `configMap` with the dashboards configuration
- Sets the `k8s-sidecar-target-directory `annotation to indicate the [k8s-sidecar](https://github.com/kiwigrid/k8s-sidecar) in which folders save the files

### CODE REVIEW

**Comment by rodrmartinez** [comment]
I applied all the dashboards to the Testing AKS cluster. Please take a look to see if it make sense to include them

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+```bash
+make dashboards-docker
+```
+
+The second option is to run this other command on the root folder:
+
+```bash
+make gen-dashboards
+```
```
Comment (giannoul) [inline README.md:15]: Is this valid? There is no Makefile in the root directory nor does the target `gen-dashboards` exist.

Comment (rodrmartinez) [reply README.md:15]: yes, I left the README from the old cre-mk. I have to update it for this new repo

**Comment by rodrmartinez** [comment]
This PR was integrated with https://github.com/ContainerSolutions/cre-mk-monitoring/pull/5

**Pull request closed without merging**